db/migrations/20211215170939-create-account.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+module.exports = {
+    up: async(queryInterface, Sequelize) => {
+        await queryInterface.createTable('Accounts', {
```

Comment:
please add a currency field to this migration
also maybe we need to add the color of the card to store it in the Accounts table

Reply:
Please add `type` field to this migration